addons/web/static/src/xml/graph.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates>
+
+    <div class="o_graph_canvas_container o_graph_renderer" t-name="web.GraphOwlRenderer" owl="1">
+
+        <t t-if="state.showNocontentHelper" t-call="web.NoContentHelper">
+            <t t-set="description" t-value="state.error_msg"/>
+        </t>
+
+        <t  t-call="web.GraphOwlTooltip">
+
+        </t>
+        <label t-if="title"><t t-esc="title"></t></label>
+        <canvas t-ref="chart-canvas">
+
+        </canvas>
+    </div>
+
+
+    <div t-name="web.GraphOwlTooltip" class="o_graph_custom_tooltip" t-ref="chart-tooltip" owl="1">
+        <table>
+            <thead>
+                <tr>
+                    <th class="o_measure"><t t-esc="state.tooltipMeasure"/></th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr t-foreach="state.tooltipItems" t-as="tooltipItem">
+                    <td>
+                        <span class="o_square" t-att-style="'background-color: ' + tooltipItem.boxColor"/>
```

Comment:
I guess you have the same template as before?
Don't forget to remove old files once done work :-)

Reply:
After attempting unsuccessfully to make that work properly with OWL, some problems - especially with the removing of tooltip lines in `_adjustTooltipHeight` (lines would not re-appear afterwards) - it was decided, with aab's advice, to use the good old `qweb.render` with the good old view. So, this tooltip view doesn't exist anymore, and I suppose I should leave the old one where it is currently?